mozilla_aws_cli/login.py

```diff
@@ -411,10 +411,9 @@ def exchange_token_for_credentials(self):
     def print_output(self):
         # TODO: Create a global config object?
         if self.credentials is not None:
-            if self.profile_name is None:
```

Comment:
We still want to retain the functionality where, by default, the tool picks a meaningful profile name to use. So I'd keep this behavior and merely change the display behavior.

Reply:
Right, it still does. It gets set just one line lower?

Reply:
Done!